frontend/src/locales/de.json

```diff
@@ -512,6 +512,7 @@
       "copied": "{source} kopiert"
     },
     "validation": {
+      "greaterThan": "{_field_} muss grösser sein als {min}",
```

Comment:
Looks not so great in the UI, can we improve this?
![grafik](https://github.com/user-attachments/assets/aaade4ca-096c-40db-97ad-b3e1cc2743ef)

Reply:
This is a separate bug in firefox for all validations: 
![image](https://github.com/user-attachments/assets/9f1ea78a-fbf0-47db-ab02-860e65bccf4c)
-> new issue

Reply:
https://github.com/ecamp/ecamp3/issues/5691